<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>241</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>611</width>
    <height>241</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>611</width>
     <height>200</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Alfa inicial</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="alfa1"/>
          </item>
          <item>
           <widget class="QLineEdit" name="alfa2"/>
          </item>
          <item>
           <widget class="QLineEdit" name="alfa3"/>
          </item>
          <item>
           <widget class="QLineEdit" name="alfa4"/>
          </item>
          <item>
           <widget class="QLineEdit" name="alfa5"/>
          </item>
          <item>
           <widget class="QLineEdit" name="alfa6"/>
          </item>
          <item>
           <widget class="QLineEdit" name="alfa7"/>
          </item>
          <item>
           <widget class="QLineEdit" name="alfa8"/>
          </item>
          <item>
           <widget class="QLineEdit" name="alfa9"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Variacion por iteracion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="var">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>N° Iteraciones búsqueda alfas</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="iter_b_alfas">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>N° Iteraciones validación alfa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="iter_v_alfas">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Tipo Simulación</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="tipo_sim"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="sim">
        <property name="text">
         <string>Simular</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
